code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -169,7 +123,9 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 
 	return 1
 
-/datum/gas_mixture/merge(datum/gas_mixture/giver)
+/datum/gas_mixture/proc/merge(datum/gas_mixture/giver)
+	//Merges all air from giver into self. Deletes giver.
+	//Returns: 1 if we are mutable, 0 otherwise

```

Comment:
```suggestion
///Merges all air from giver into self. Deletes giver. Returns: TRUE if we are mutable, FALSE otherwise
/datum/gas_mixture/proc/merge(datum/gas_mixture/giver)
```
Do you know how to document the procs?